flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
upstream has an aarch64 build. any reason to exclude it here?

Reply:
Ok, I think need to add.
How to determine the arch of a package which need download in the source section in the YAML file?
I mean the link: `https://github.com/Figma-Linux/figma-linux/releases/download/v0.7.2/figma-linux_0.7.2_linux_amd64.deb`

Reply:
You can define `only-arches` also in the specific source file (along with checksum etc).